packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -113,10 +113,20 @@ export default /* @ngInject */ ($stateProvider) => {
       getUpscaleHref: /* @ngInject */ ($state) => () =>
         $state.href('vps.detail.upscale'),
 
-      goToUpgrade: /* @ngInject */ ($state) => (upgradeType) =>
-        $state.go(`vps.detail.dashboard.configuration.upgrade`, {
+      goToUpgrade: /* @ngInject */ ($state, stateVps) => (upgradeType) => {
+        let from = 0;
+        if (upgradeType === 'memory') {
+          from = stateVps.model[upgradeType] / 1024;
+        } else if (upgradeType === 'storage') {
+          from = stateVps.model.disk;
+        }
+        const to = from * 2;
```

Comment:
Always the case ?

Reply:
Seems to be : https://github.com/ovh/manager/blob/develop/packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/routing.js#L132-L141